anybox/recipe/odoo/base.py

```diff
@@ -1618,6 +1629,17 @@ def finalize_addons_paths(self, check_existence=True):
                 assert os.path.isdir(path), (
                     "Not a directory: %r (aborting)" % path)
 
+        self.addons_paths = [
+            os.path.relpath(
+                addons_path,
+                os.path.join(
+                    self.bin_dir,
+                    'openerp' if 'openerp' in addons_path else 'odoo'
+                ))
```

Comment:
In my configuration this goes wrong: 

- `self.bin_dir` is `$BUILDOUT/bin`
- `addons_path` is `$BUILDOUT/parts/some_repo` and as such does not contain either `openerp` or `odoo`.
- Total expression comes to `$BUILDOUT/bin/odoo`
- Relative path becomes `../../parts/some_repo` and is written to `$BUILDOUT/etc/odoo.cfg`
- Odoo is run from `$BUILDOUT/parts/odoo/openerp`, tries to find addons in $BUILDOUT/parts/parts/some_repo`, fails.

Reply:
@sebastiken seems your patch doesn't work for everyone, you you have a look at https://github.com/anybox/anybox.recipe.odoo/pull/104#pullrequestreview-63599099?

Reply:
Ok @hbrunn I will see this using the configuration proposed by @thomaspaulb